test/integration/test_integration.py

```diff
@@ -104,7 +105,13 @@
 def _int8wo_api(mod):
     if TORCH_VERSION_AT_LEAST_2_4:
         quantize_(mod, int8_weight_only(), set_inductor_config=False)
-        if not TORCH_VERSION_AT_LEAST_2_5:
+        if (
```

Comment:
wow condition is becoming pretty complicated..

Reply:
yean, ensure as much as test combination has been covered.